src/components/PriceChart/PriceChart.tsx

```diff
@@ -70,6 +70,8 @@ export const PriceChart: React.FC<PriceChartArgs> = props => {
     selectPriceHistoryByAssetTimeframe(state, assetId, timeframe),
   )
 
+  console.log({ priceData })
```

Comment:
ser

Reply:
I don't know what you're talking about, must be a stale diff or something

![monkeylookaway](https://media1.giphy.com/media/v1.Y2lkPTc5MGI3NjExOWJkZ2RqcnNybXFsNHpjcXE4bWx4cTI0ZnVhZzk2dDB3ancyY2lrMSZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/H5C8CevNMbpBqNqFjl/giphy.webp)

(d8b289610)